### PR TITLE
Introduce sessions to ntt report

### PR DESCRIPTION
**Pull request by 0xmojo**

### CODE REVIEW

**Review by 5nord — COMMENTED** [review]
I am wondering, why you did not just extend DB with Sessions, but created second Results struct and some conversion function? I fail to see the benefit.

**Comment by 0xmojo** [comment]
> I am wondering, why you did not just extend DB with Sessions, but created second Results struct and some conversion function? I fail to see the benefit.
> 
> You could promote the Max-Field to methods. For example:
> 
> ```go
> type  DB struct {
>     Version string
>     Sessions []Session
> }
> 
> // Runs returns a slice with all runs from all sessions
> func (db *DB) Runs() []Run {
>     var runs []Run
>     for i, r := range db.Sessions {
>         runs = append(runs, r.Runs)
>     }
>     return runs
> }
> 
> func (db *DB) MaxLoad() float64 {
>     if len(db.Sessions) == 0 {
>        return 0
>     }
>     max := db.Sessions[0].MaxLoad
>     for _, s := range db.Sessions {
>         if s.MaxLoad > max {
>             max = s.MaxLoad
>     }
>     return max
> }
> ```

Yes i know :) I actually did this in my first approach, but it was a lot of additional work to be done, because then the whole parsing tree from db root node did not work anymore. So i ditched it and thought it would work if we only add "Expected Verdict" as an additional field to the Run struct and just transform the new format to the old format. Then it should be enought to use the expecgted verdict in the FinalVerdict Function. 

To give the Db struct a Runs function might also work. I will check this on monday.

File: internal/results/results.go [hunk 1]
```diff
@@ -9,12 +9,55 @@ import (
...
+	for _, s := range db.Sessions {
+		if s.MaxLoad > max {
+			max = s.MaxLoad
+		}
+	}
+	return max
+}
+
+type Session struct {
+	id              string
```
Comment (5nord) [inline internal/results/results.go:56]: Is there a reason you made `id` private?

**Review by 5nord — APPROVED** [review]
Nice. Concise. Looks good :)

On little remark though: Please describe `FixedTests` in our help message and documentation.

**Review by moosq — APPROVED** [review]

**Pull request merged**